command/agent/cache/lease_cache_test.go

```diff
@@ -711,13 +712,18 @@ func setupBoltStorage(t *testing.T) (tempCacheDir string, boltStorage *cachebolt
 }
 
 func TestLeaseCache_PersistAndRestore(t *testing.T) {
-	// Emulate 4 responses from the api proxy. The first two use the auto-auth
-	// token, and the last two use another token.
+	// Emulate responses from the api proxy. The first two use the auto-auth
+	// token, and the others use another token.
```

Comment:
Since we are relying on `[]cacheTests` and `[]responses` having equal lengths it might be good to document that here.